components/progressbar/stories/progressbar.stories.js

```diff
@@ -51,18 +83,110 @@ export default {
+ * 
+ * When the label is too long for the available horizontal space, it wraps to form another line. The value is always shown in full and never wraps or truncates.
+ */
 export const Default = ProgressBarGroup.bind({});
 Default.args = {
-	label: "Loading",
 	value: 50,
 };
 
+/**
+ * Progress bars come in four different sizes: small, medium, large, and extra-large. The medium size is the default and most frequently used option.
+ */
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withBorder: false,
+	withHeading: false,
+	...args, 
+}, context);
+Sizing.args = Default.args;
+Sizing.tags = ["!dev"];
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * Progress bars can represent either determinate or indeterminate progress. Use a determinate progress bar when progress can be calculated against a specific goal (e.g., downloading a file of a known size). Use an indeterminate progress bar when progress is happening, but the time or effort to completion can’t be determined (e.g., attempting to reconnect to a server).
+ */
+export const Indeterminate = IndeterminateGroup.bind({});
+Indeterminate.args = {
+	value: 50,
+};
+Indeterminate.tags = ["!dev"];
+Indeterminate.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The minimum width of a progress bar is 48px and the maximum width of a progress bar is 768px, for both desktop and mobile platform scale. Smaller progress bars should only be used in places where it’s not necessary to have a label.
+ */
+export const CustomWidth = Template.bind({});
+CustomWidth.storyName = "Custom width";
+CustomWidth.args = {
+	value: 83,
+	customWidth: 400,
+};
+CustomWidth.tags = ["!dev"];
+CustomWidth.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The example below shows a progress bar without its label. Progress bars typically have a label, but in rare cases where context is sufficient and an accessibility expert has reviewed the design, the label could be undefined.
+ */
+export const WithoutLabel = Template.bind({});
+WithoutLabel.storyName = "Without label";
+WithoutLabel.args = {
+	label: "",
+	value: 50,
+	showValueLabel: false,
+};
+WithoutLabel.tags = ["!dev"];
+WithoutLabel.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The example below shows a progress bar without its value label. This value label works in conjunction with the label, and should not be displayed if the label itself is not displayed.
+ */
+export const WithoutValue = Template.bind({});
+WithoutValue.storyName = "Without value";
+WithoutValue.args = {
+	value: 83,
+	showValueLabel: false,
+};
+WithoutValue.tags = ["!dev"];
+WithoutValue.parameters = {
+	chromatic: {
+		modes: disableDefaultModes
+	},
+};
+
+export const StaticWhiteDocs = Template.bind({});
```

Comment:
```suggestion
export const StaticWhiteDocs = Default.bind({});
```

If you map this to the Default template, you'll get the full testing grid validated in the static white context.